spring-rabbit/src/main/java/org/springframework/amqp/rabbit/listener/SimpleMessageListenerContainer.java

```diff
@@ -835,7 +835,13 @@ protected void doShutdown() {
 	private boolean isActive(BlockingQueueConsumer consumer) {
 		Boolean consumerActive;
 		synchronized(consumersMonitor) {
-			consumerActive = this.consumers != null && this.consumers.get(consumer);
```

Comment:
Yeah... That is really nasty bug from our side: do not care about outboxing for `null` :cry: .
Anyway, good to know and thanks for fix!